src/server/pps/server/worker_rc.go

```diff
@@ -479,7 +475,7 @@ func hashAuthToken(token string) string {
 	return base64.RawURLEncoding.EncodeToString(h[:])
 }
 
-func (a *apiServer) getWorkerOptions(pipelineInfo *pps.PipelineInfo) (*workerOptions, error) {
+func (a *apiServer) getWorkerOptions(ctx context.Context, pipelineInfo *pps.PipelineInfo) (*workerOptions, error) {
```

Comment:
am I missing something or is the context unused? (I guess the k8s client uses some implied context?)

Reply:
no, you're right, it's unused - this is just an artifact of the port from `master` (and part of why the cherry-pick wasn't trivial) - we've changed kubernetes client versions in `master`, and the new version uses contexts on all the calls.  This parameter is extraneous, but I'm inclined to just leave it here to reduce the delta between `master` and `2.0.x`.